### PR TITLE
LPS-165932 [PRM] Create MDF Chart

### PR DESCRIPTION
**Pull request by claricebrasil**

### CODE REVIEW

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/dashboard-collection/fragments/mdf-request-chart/index.js [hunk 1]
```diff
@@ -10,132 +10,60 @@
...
+	};
 
-	const sortedItemsClain = Object.entries(mdf.ProgressClain);
+	useEffect(() => {
+		getMDFRequests();
 
-	const totalRequest =
-		mdf.ProgressMdf.approved.qtd + mdf.ProgressMdf.pending.qtd;
+		// eslint-disable-next-line react-hooks/exhaustive-deps
+	}, []);
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/dashboard-collection/fragments/mdf-request-chart/index.js:56]:
```suggestion
	}, []);
```

Comment (jairmedeiros) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/dashboard-collection/fragments/mdf-request-chart/index.js:56]: Let's avoid to use this eslint

**Review by jairmedeiros — CHANGES_REQUESTED** [review]

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/dashboard-collection/fragments/mdf-request-chart/index.js (hunk 1 above)
Comment (claricebrasil) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/dashboard-collection/fragments/mdf-request-chart/index.js]: Done

**Comment by claricebrasil** [comment]
ci:forward

**Comment by thaynam** [comment]
ci:forward

**Pull request closed without merging**